app/views/projects/show.html.erb

```diff
@@ -122,6 +122,10 @@
                                                                         url: storage_report_project_path(@project)} %>
         <% end %>
 
+        <% if @project.is_taggable?%>
+          <%= render :partial=>"assets/tags_box", locals: {resource: @project}, :no_tags_message=>"Add tags (comma separated) ..." -%>
```

Comment:
same as before ....

update to use the more modern json like format, i.e

partial: 'assets/tags_box', locals: { resource: resource } ...